lib/action_cable_client.rb

```diff
@@ -91,7 +91,7 @@ def received
   #     # do things after the client is connected to the server
   #   end
   def connected
-    _websocket_client.onopen do
```

Comment:
So, if I understand you correctly, the onopen event doesn't matter anymore, because we need to wait until the the action cable server is _ready_ for us to subscribe to it, and before this PR change, there were situations in which the action cable server was not ready to accept subscriptions immediately after a connection was established?

Reply:
Exactly. The state transition on the server is pretty quick so it doesn't occur often, but when it does the subscription message from the client is "swallowed". Leaving the server waiting for a "subscribe" and the client waiting for the subscription confirmation.